<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTK-Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
    background-color: #252526;
    border-bottom: 1px solid #404040;
    color: #e0e0e0;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;;
    font-size: 9pt;
    padding: 1px;
}

QMenuBar::item {
    background-color: transparent;
    color: #e0e0e0;
    padding: 6px 16px;
    margin: 0px 2px;
    border-radius: 3px;
}

QMenuBar::item:selected {
    background-color: #3a6ea5;
    color: #ffffff;
}

QMenuBar::item:pressed {
    background-color: #2a5685;
    color: #ffffff;
}

QMenuBar::item:disabled {
    color: #666666;
}

/* 菜单下拉部分的样式 */
QMenu {
    background-color: #2b2b2b;
    border: 1px solid #404040;
    color: #e0e0e0;
    font-size: 9pt;
    padding: 4px;
}

QMenu::item {
    background-color: transparent;
    color: #e0e0e0;
    padding: 6px 24px 6px 16px;
    margin: 2px 4px;
    border-radius: 3px;
}

QMenu::item:selected {
    background-color: #3a6ea5;
    color: #ffffff;
}

QMenu::item:disabled {
    color: #666666;
}

QMenu::separator {
    height: 1px;
    background: #404040;
    margin: 4px 8px;
}

QMenu::indicator:checked {
    background-color: #3a6ea5;
    border: 2px solid #5a8ec9;
    border-radius: 2px;
}

QStatusBar{
       background-color: #252526;
}

 QMessageBox {
            background-color: #2b2b2b;
            color: #ffffff;
            font-family: &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;;
        }
        QMessageBox QLabel {
            color: #e0e0e0;
            font-size: 14px;
        }
        QMessageBox QPushButton {
            background-color: #404040;
            color: #ffffff;
            border: 1px solid #555555;
            border-radius: 4px;
            padding: 8px 16px;
            min-width: 80px;
            font-size: 13px;
        }
        QMessageBox QPushButton:hover {
            background-color: #505050;
            border-color: #666666;
        }
        QMessageBox QPushButton:pressed {
            background-color: #303030;
        }
        QMessageBox QPushButton:focus {
            border: 1px solid #0078d4;
        }


*{
   color:rgb(198, 198, 198)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:#2d2d30;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#leftWidget{
  border:1px solid #4a4a4a;
background-color:#2d2d2d;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="leftLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                </property>
                <property name="text">
                 <string>DICOM浏览器</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    border: 2px solid #4a4a4a;   
    color: #333;                 
}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-image: url(:/icon/assets/imgs/搜索.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="ProjectTree">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTreeView {
    border: 0px solid #8a8a8a30;   
    color: #d4d4d4;          
    background-color: rgb(37, 37, 38);
}</string>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="vtkContainer"/>
      </widget>
      <widget class="QWidget" name="rightWidget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#rightWidget{
 border:1px solid #4a4a4a;
background-color:#2d2d2d;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="rightLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>影像标签</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="topMargin">
                 <number>6</number>
                </property>
                <property name="bottomMargin">
                 <number>6</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit {
    border: 2px solid #4a4a4a;   
    color: #333;                 
}</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-image: url(:/icon/assets/imgs/搜索.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tagTab">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/* 整体背景 */
QWidget {
    background-color: #252526;
    color: #d4d4d4;
    selection-background-color: #007acc;
}

/* QTabWidget 样式 */
QTabWidget::pane {
    border: 1px solid #333333;
    background-color: rgb(255, 85, 0);
}

QTabBar::tab {
    background-color: #2d2d30;
    color: #d4d4d4;
    padding: 6px 12px;
    border: 1px solid #333333;
    border-bottom: none;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    margin-right: 2px;
}

QTabBar::tab:hover {
    background-color: #3e3e42;
}

QTabBar::tab:selected {
    background-color: #1e1e1e;
    color: #ffffff;
}


/* QTableView 样式 */
QTableView {
    background-color: #1e1e1e;
    alternate-background-color: #2a2d2e;
    gridline-color: #3c3c3c;
    selection-background-color: #094771;
    selection-color: #ffffff;
    border: 1px solid #333333;
}

QHeaderView::section {
    background-color: #2d2d30;
    color: #cccccc;
    border: 1px solid #3c3c3c;
    padding: 4px;
}

QTableCornerButton::section {
    background-color: #2d2d30;
    border: 1px solid #3c3c3c;
}

/* 滚动条（可选） */
QScrollBar:vertical {
    background: #252526;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #4e4e50;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #5e5e60;
}

QScrollBar:horizontal {
    background: #252526;
    height: 10px;
    margin: 0px;
}
QScrollBar::handle:horizontal {
    background: #4e4e50;
    border-radius: 5px;
}
QScrollBar::handle:horizontal:hover {
    background: #5e5e60;
}
</string>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::South</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="patientTab">
             <attribute name="title">
              <string>病人信息</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QTableView" name="patientTagTable">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="allTab">
             <attribute name="title">
              <string>所有信息</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTableView" name="allTagTable">
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionCloseCurrent"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionCloseCurrent">
   <property name="text">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
